tests/test_buffer.py

```diff
@@ -0,0 +1,215 @@
+
+        # remove lossless PG 3-4 and 6 on interface
+        dvs.runcmd("config interface buffer priority-group lossless remove Ethernet0 3-4")
+        dvs.runcmd("config interface buffer priority-group lossless remove Ethernet0 6")
+
+    @pytest.mark.skip(reason="Traditional buffer model runs on VS image currently.")
+    def test_headroomOverride(self, dvs, testlog):
+        self.setup_db(dvs)
+
+        # configure lossless PG 3-4 on interface
+        dvs.runcmd("config interface buffer priority-group lossless add Ethernet0 3-4")
+
+        # Configure static profile
+        dvs.runcmd("config buffer profile add test --xon 18432 --xoff 16384")
+        self.app_db.wait_for_exact_match("BUFFER_PROFILE_TABLE", "test",
+                        { "pool" : "[BUFFER_POOL_TABLE:ingress_lossless_pool]",
+                          "xon" : "18432",
+                          "xoff" : "16384",
+                          "size" : "34816",
+                          "dynamic_th" : "0"
+                          })
+
+        dvs.runcmd("config interface cable-length Ethernet0 " + self.cableLenTest1)
+        expectedProfile = self.make_lossless_profile_name(self.originalSpeed, self.cableLenTest1)
+        self.app_db.wait_for_entry("BUFFER_PROFILE_TABLE", expectedProfile)
+        self.app_db.wait_for_field_match("BUFFER_PG_TABLE", "Ethernet0:3-4", {"profile": "[BUFFER_PROFILE_TABLE:" + expectedProfile + "]"})
+
+        dvs.runcmd("config interface buffer priority-group lossless set Ethernet0 3-4 test")
+        self.app_db.wait_for_field_match("BUFFER_PG_TABLE", "Ethernet0:3-4", {"profile": "[BUFFER_PROFILE_TABLE:test]"})
+        dvs.runcmd("config interface buffer priority-group lossless add Ethernet0 6 test")
+        self.app_db.wait_for_field_match("BUFFER_PG_TABLE", "Ethernet0:6", {"profile": "[BUFFER_PROFILE_TABLE:test]"})
+
+        dvs.runcmd("config interface buffer priority-group lossless remove Ethernet0")
+        self.app_db.wait_for_deleted_entry("BUFFER_PG_TABLE", "Ethernet0:3-4")
+        self.app_db.wait_for_deleted_entry("BUFFER_PG_TABLE", "Ethernet0:6")
+
+        dvs.runcmd("config interface buffer priority-group lossless add Ethernet0 3-4")
+        self.app_db.wait_for_field_match("BUFFER_PG_TABLE", "Ethernet0:3-4", {"profile": "[BUFFER_PROFILE_TABLE:" + expectedProfile + "]"})
+
+        dvs.runcmd("config interface cable-length Ethernet0 " + self.originalCableLen)
+        self.app_db.wait_for_deleted_entry("BUFFER_PROFILE_TABLE", expectedProfile)
+        expectedProfile = self.make_lossless_profile_name(self.originalSpeed, self.originalCableLen)
+        self.app_db.wait_for_entry("BUFFER_PROFILE_TABLE", expectedProfile)
+        self.app_db.wait_for_field_match("BUFFER_PG_TABLE", "Ethernet0:3-4", {"profile": "[BUFFER_PROFILE_TABLE:" + expectedProfile + "]"})
+
+        # remove lossless PG 3-4 on interface
+        dvs.runcmd("config interface buffer priority-group lossless remove Ethernet0 3-4")
+
```

Comment:
Please add testcase for mtu change, port admin status change and non default alpha as well?

Reply:
Done